data/INRA/dimensions.ttl

```diff
@@ -0,0 +1,45 @@
+bdg:date a qb:DimensionProperty;
+  rdfs:label "Date";
+  qb:concept sdmx-concept:timePeriod;
+  rdfs:range xsd:date;
+  dct:description "Date of the observation".
+
+
+bdg:humanSensor a qb:DimensionProperty;
+  rdfs:label "Human sensor";
+  qb:concept sdmx-concept:collMethod;
+  rdfs:range sosa:Sensor;
+  dct:description "Human that took the observation".
+
+
+bdg:variety a qb:DimensionProperty;
```

Comment:
this is also a `qb:CodedProperty` and you should tell it which codelist to use - `qb:Codelist <variety>`

Reply:
Also all of the other dimensions here are coded properties